.travis.yml

```diff
@@ -12,8 +12,7 @@ matrix:
     env: PUPPET_GEM_VERSION="~> 3.0"
   - rvm: 2.1.5
     env: PUPPET_GEM_VERSION="~> 4.0"
-  - rvm: 1.9.3
-    env: PUPPET_GEM_VERSION="3.0.0"
+  - rvm: 2.0.0
+    env: PUPPET_GEM_VERSION="~> 3.0" FUTURE_PARSER="yes"
```

Comment:
What settings should they be?  This is the default travis set.  Would you like a specific test for 3.0.0?